Source/WebCore/accessibility/atspi/AccessibilityObjectTextAtspi.cpp

```diff
@@ -276,8 +276,11 @@ String AccessibilityObjectAtspi::text() const
     m_hasListMarkerAtStart = false;
 
     if (m_coreObject->roleValue() == AccessibilityRole::ColorWell) {
-        auto color = convertColor<SRGBA<float>>(m_coreObject->colorValue()).resolved();
-        GUniquePtr<char> colorString(g_strdup_printf("rgb %7.5f %7.5f %7.5f 1", color.red, color.green, color.blue));
+        auto color = convertColor<SRGBA<float>>(m_coreObjeWct->colorValue()).resolved();
+        GUniquePtr<char> colorString(color.alpha == 1.0f
+            ? g_strdup_printf("rgb %7.5f %7.5f %7.5f 1", color.red, color.green, color.blue)
+            : g_strdup_printf("rgb %7.5f %7.5f %7.5f %7.5f", color.red, color.green, color.blue, color.alpha)
```

Comment:
The linter doesn't seem to like the indentation on the second branch of the ternary (line 282) but I'm not sure why? It seems pretty standard?